### ISSUE TITLE
Position of Parameters in documentation of a function (handling of @remarks)

### ISSUE THREAD

**Issue by petrvilim** [tgreyuk/typedoc-plugin-markdown#451]
Thank you for your work!

Here's documentation for a class method generated by typedoc without the markdown plugin. Notice that ***Parameters*** and ***Returns*** are at the top of the page, just after the brief description of the function and before ***Remarks***:

![image](https://github.com/tgreyuk/typedoc-plugin-markdown/assets/74963906/f03efc0a-452f-4dc5-a5fb-a73f41440a70)

With typedoc-plugin-markdown, ***Parameters*** are at the bottom, regardless of the `@remarks` tag:

![image](https://github.com/tgreyuk/typedoc-plugin-markdown/assets/74963906/ebaec1cc-03eb-4f0e-bb1c-342e3fa1b439)

Is there a way to move ***Parameters*** up? Am I missing something obvious?

I'm using TypeDoc 0.24.8 and typedoc-plugin-markdown 3.15.3. The screenshot above is from docusaurus 2.3.1 with docusaurus-plugin-typedoc 0.19.2.

The documentation in the source file looks like this:

```
  /** 
  * Creates an integer expression for the end of an interval variable.
  * 
  * @remarks
  * 
  * If the interval is absent then the resulting expression is also absent.
  * 
  * @example
  * 
  * In the following example we constraint interval variable `y` to start after end of `y` with a delay at least 10. In addition we constrain length of `x` to be less or equal than length of `y`.
  * 
  * ```ts
  * let model = new CP.Model;
  * let x = model.intervalVar({ name: "x", ... });
  * let y = model.intervalVar({ name: "y", ... });
  * model.constraint(model.endOf(x).plus(10).le(model.startOf(y)));
  * model.constraint(model.lengthOf(x).le(model.lengthOf(y)));
  * ```
  * 
  * When `x` or `y` is _absent_ then value of both constraints above is _absent_ and therefore they are satisfied.
  * 
  * @see {@link IntervalVar.end} is equivalent function on {@link IntervalVar}.
  * @see Function {@link Model.endOr} is a similar function that replaces value _absent_ by a constant.
  *  */
  endOf(interval: IntervalVar): IntExpr { 
```

Thank you!

**Comment by tgreyuk** [comment]
Thanks - have updated to follow pattern of the html theme. Available in typedoc-plugin-markdown@3.15.4.

**Comment by petrvilim** [comment]
Thank you, that was really fast! It works like a charm.
